src/config/backpack/base.php

```diff
@@ -68,11 +68,19 @@
 
     /*
     |--------------------------------------------------------------------------
-    | User Model
+    | Authentication
     |--------------------------------------------------------------------------
     */
 
     // Fully qualified namespace of the User model
     'user_model_fqn' => '\App\User',
```

Comment:
@jorenvanhee , I think as long as we don't eliminate ```user_model_fqn``` at this time, this is a non-breaking change. Right?